user_token.go

```diff
@@ -50,12 +50,14 @@ type UserToken struct {
 	Description string    `jsonapi:"attr,description"`
 	LastUsedAt  time.Time `jsonapi:"attr,last-used-at,iso8601"`
 	Token       string    `jsonapi:"attr,token"`
+	ExpiredAt   time.Time `jsonapi:"attr,expired-at,iso8601"`
 }
 
 // UserTokenCreateOptions the options for creating a user token.
 type UserTokenCreateOptions struct {
-	// Optional: Description of the token
-	Description string `jsonapi:"attr,description,omitempty"`
+	// Optional: Description of the token and its expiration date
+	Description string    `jsonapi:"attr,description,omitempty"`
+	ExpiredAt   time.Time `jsonapi:"attr,expired-at,iso8601,omitempty"`
```

Comment:
❓ how does this behave if the caller doesnt specify an `ExpiredAt`?

I am fuzzy on how `omitempty` works with `time.Time`, and a little brain foggy right now, but I wonder if this is left out if it will:
- Default to the default "time" if such a thing exists...which would probably be wrong and something like `0000-00-00 00:00:00` or the unix epoch or something.
- Just be omitted from the request, which would be correct behaviour
If it is the first one, I think we may need to make this a pointer to a time.Time (`*time.Time`)